### PR TITLE
Modified scripts Aug 30

### PR DESCRIPTION
**Pull request by manjunath-grl**
Modified automation script
TC-CC-*
TC-CHANNEL-1.6
TC-DGGEN-1.1
TC-DGTHREAD-1.1
TC-DRLK-1.1
TC-DRLK-2.4
TC-DRLK-2.9
TC-PRS-1.1
TC-PS-1.1
TC-TSTAT-1.1
TC-SWTCH-1.1
TC-WNCV-1.1
TC-WNCV-2.1
TC-WNCV-2.1
TC-WNCV-3.*
Modified Manual tests
TC-IDM-*
TC-ACl-2.5
TC-ACT-2.1
TC-OO-3.1
Added auto generated files

Tested
Attached execution logs:
[Execution_log.txt](https://github.com/project-chip/connectedhomeip/files/9455693/Execution_log.txt)

### CODE REVIEW

File: src/app/tests/suites/certification/Test_TC_CC_3_1.yaml [hunk 1]
```diff
@@ -207,13 +207,22 @@ tests:
...
     - label: "TH reads CurrentHue attribute from DUT"
-      PICS: CC.S.F00 && CC.S.A0000
-      command: "readAttribute"
-      attribute: "CurrentHue"
-      response:
-          constraints:
-              minValue: 0
-              maxValue: 15
+      PICS: CC.S.F00 && CC.S.A0000 && PICS_SKIP_SAMPLE_APP
+      verification: |
```
Comment (andy31415) [inline src/app/tests/suites/certification/Test_TC_CC_3_1.yaml:211]: why was this changed to userprompt? Is reading attributes with min/max not sufficient? Why?

Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_CC_3_1.yaml:211]: @andy31415 The problem here is that the values count down from 60 -> 0 -> 135, as per the test plan after 10 seconds that values needs to be exactly 0 however this does not happen in real world scenarios as the DUT/Delay command behaviour changes across setups. We had previously added a range check 0-15 [+15% tolerance around the expected value of 0] but -15% is not possible as the expected value is 0 and we currently cannot specify the range checks in descending order.
Please refer this issue:  [416](https://github.com/CHIP-Specifications/chip-test-scripts/issues/416)

File: src/app/tests/suites/certification/Test_TC_WNCV_1_1.yaml [hunk 2]
```diff
@@ -45,14 +45,60 @@ tests:
...
+              hasMasksSet: [0x2]
+
+    - label: "Given WNCV.S.F02(PA_LF) ensure featuremap has the correct bit set"
+      PICS: WNCV.S.F02
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x4]
```
Comment (jmeg-sfy) [inline src/app/tests/suites/certification/Test_TC_WNCV_1_1.yaml:83]:
```suggestion
              hasMasksSet: [0x5]
```

File: src/app/tests/suites/certification/Test_TC_WNCV_1_1.yaml [hunk 3]
```diff
@@ -45,14 +45,60 @@ tests:
...
+              hasMasksSet: [0x8]
+
+    - label: "Given WNCV.S.F04(PA_TL) ensure featuremap has the correct bit set"
+      PICS: WNCV.S.F04
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x10]
```
Comment (jmeg-sfy) [inline src/app/tests/suites/certification/Test_TC_WNCV_1_1.yaml:101]:
```suggestion
              hasMasksSet: [0x12]
```

File: src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml [hunk 4]
```diff
@@ -176,22 +174,14 @@ tests:
               minValue: 5
               maxValue: 21
 
-    ### Depending on the DUT specs we might prefer a arguments here
-    - label: "3a2: DUT updates its attributes"
```
Comment (jmeg-sfy) [inline src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml:180]: keep this step

File: src/app/tests/suites/certification/Test_TC_WNCV_3_2.yaml [hunk 5]
```diff
@@ -176,22 +174,14 @@ tests:
               minValue: 10
               maxValue: 42
 
-    ### Depending on the DUT specs we might prefer a arguments here
-    - label: "3a2: DUT updates its attributes"
-      cluster: "DelayCommands"
```
Comment (jmeg-sfy) [inline src/app/tests/suites/certification/Test_TC_WNCV_3_2.yaml:181]: keep this step

File: src/app/tests/suites/certification/Test_TC_WNCV_3_2.yaml [hunk 6]
```diff
@@ -176,22 +174,14 @@ tests:
...
-      command: "WaitForMs"
-      arguments:
-          values:
-              - name: "ms"
-                value: 2000
```
Comment (jmeg-sfy) [inline src/app/tests/suites/certification/Test_TC_WNCV_3_2.yaml:186]: increase to 3000ms if still  failing

File: src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml [hunk 7]
```diff
@@ -176,22 +174,14 @@ tests:
...
-      cluster: "DelayCommands"
-      command: "WaitForMs"
-      arguments:
-          values:
-              - name: "ms"
-                value: 2000
```
Comment (jmeg-sfy) [inline src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml:186]: increase to 3000ms if still failing

File: src/app/tests/suites/certification/Test_TC_WNCV_1_1.yaml (hunk 2 above)
Comment (jmeg-sfy) [reply src/app/tests/suites/certification/Test_TC_WNCV_1_1.yaml:83]: PA_LF must have LF set as well

File: src/app/tests/suites/certification/Test_TC_WNCV_1_1.yaml (hunk 3 above)
Comment (jmeg-sfy) [reply src/app/tests/suites/certification/Test_TC_WNCV_1_1.yaml:101]: PA_TL must have TL set as well

File: src/app/tests/suites/certification/ci-pics-values [hunk 8]
```diff
@@ -61,8 +61,8 @@ MANUAL_FLOW_CHANGE=1
 WNCV.S.F00=1
 WNCV.S.F02=1
 WNCV.S.F01=1
-WNCV.S.F04=1
-WNCV.S.F03=1
+WNCV.S.F04=0
```
Comment (jmeg-sfy) [inline src/app/tests/suites/certification/ci-pics-values:64]: Why removing these PICS ?

Comment (jmeg-sfy) [reply src/app/tests/suites/certification/ci-pics-values:64]: WNCV.S.F03=0 is correct
but WNCV.S.F04=1 should stay at 1

File: src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml [hunk 9]
```diff
@@ -176,22 +174,14 @@ tests:
               minValue: 5
```
Comment (jmeg-sfy) [inline src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml:174]: hasMasksSet:  should be used here instead of ranges

File: src/app/tests/suites/certification/Test_TC_WNCV_3_2.yaml [hunk 10]
```diff
@@ -176,22 +174,14 @@ tests:
               minValue: 10
```
Comment (jmeg-sfy) [inline src/app/tests/suites/certification/Test_TC_WNCV_3_2.yaml:174]: hasMasksSet: should be used here instead of ranges

**Review by jmeg-sfy — CHANGES_REQUESTED** [review]

File: src/app/tests/suites/certification/Test_TC_WNCV_1_1.yaml (hunk 2 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_WNCV_1_1.yaml:83]: @jmeg-sfy We are checking each bit in a different step as different PICS are applicable. line no 58 checks for WNCV.S.F00(LF) flag and line no 76 checks for WNCV.S.F02(PA_LF)
image

File: src/app/tests/suites/certification/Test_TC_WNCV_1_1.yaml (hunk 3 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_WNCV_1_1.yaml:101]: @jmeg-sfy We are checking each bit in a different step as different PICS are applicable. line no 67 checks for WNCV.S.F01(TL) flag and line no 94 checks for WNCV.S.F04(PA_TL)
image

File: src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml (hunk 4 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml:180]: I will update

File: src/app/tests/suites/certification/Test_TC_WNCV_3_2.yaml (hunk 5 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_WNCV_3_2.yaml:181]: i will update

File: src/app/tests/suites/certification/Test_TC_WNCV_3_2.yaml (hunk 6 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_WNCV_3_2.yaml:186]: Sure i will update

File: src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml (hunk 7 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml:186]: Sure i will update

File: src/app/tests/suites/certification/ci-pics-values (hunk 8 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/ci-pics-values:64]: I will check it and update

File: src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml (hunk 7 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml:186]: even though the delay is increased its still failing.

File: src/app/tests/suites/certification/Test_TC_WNCV_3_2.yaml (hunk 6 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_WNCV_3_2.yaml:186]: Even though the delay is increased its still failing

File: src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml (hunk 9 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml]: Updated

File: src/app/tests/suites/certification/Test_TC_WNCV_3_2.yaml (hunk 10 above)
Comment (manjunath-grl) [reply src/app/tests/suites/certification/Test_TC_WNCV_3_2.yaml]: Updated please verify it once.

File: src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml (hunk 7 above)
Comment (jmeg-sfy) [reply src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml:186]: point me to execution logs please

**Review by jmeg-sfy — APPROVED** [review]

**Review by andy31415 — APPROVED** [review]
